lib/std/crypto/benchmark.zig

```diff
@@ -109,7 +109,7 @@ pub fn benchmarkKeyExchange(comptime DhKeyExchange: anytype, comptime exchange_c
     {
         var i: usize = 0;
         while (i < exchange_count) : (i += 1) {
-            try DhKeyExchange.scalarmult(&out, out, in);
+            out = try DhKeyExchange.scalarmult(out, in);
```

Comment:
beware https://github.com/ziglang/zig/issues/6216

Reply:
In that context, it doesn't matter much. But I've updated the benchmark to take care of this and change the keys after each round by the way. Thanks!